akka-docs/src/main/paradox/stream/operators/Source-or-Flow/groupBy.md

```diff
@@ -20,7 +20,7 @@ Note: If `allowClosedSubstreamRecreation` is set to `true` substream completion
 elements are subject to race-conditions. If elements arrive for a stream that is in the process
 of closing these elements might get lost.
 
-@@@ warning
+Note: groupBy operator is used to partition the stream based on an identifier which is not same as using async boundary in the stream which is used for running the same stream concurrently. If a stream can be exclusively partitioned, it can be executed efficiently, by maximizing the parallel processing using groupBy operator.
```

Comment:
Should still have the warning block start fort the next paragraph:

```suggestion
Note: groupBy operator is used to partition the stream based on an identifier which is not same as using async boundary in the stream which is used for running the same stream concurrently. If a stream can be exclusively partitioned, it can be executed efficiently, by maximizing the parallel processing using groupBy operator.

@@@ warning
```

I think the added text should not only mention that it doesn't cause async boundaries but also mention how you can explicitly do that (something like `groupBy(..., ...).via(someFlow.async).mergeSubstreams`)

Reply:
@johanandren : do you think the example should be on this page instead : https://doc.akka.io/docs/akka/current/stream/stream-parallelism.html ?
this page explains parallelism but doesn't explains concurrency, and it can be confusing to know the difference.

for the example, 
`groupBy(..., ...).via(someFlow.async).mergeSubstreams`  why do we want to show usage of both in same example ? I'm not sure if it helps further.

Reply:
I don't necessarily think we need to show it in the example but I think the text should mention it (and possibly include a minimal snippet showing where the async would go to make each sub stream an async island)